packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -663,6 +671,86 @@ export default class GoogleCalendarService implements Calendar {
     }
   }
 
+  async getMeetParticipants(videoCallUrl: string | null): Promise<ParticipantWithEmail[][]> {
+    const { token } = await this.oAuthManagerInstance.getTokenObjectOrFetch();
+    if (!token) {
+      throw new Error("Invalid grant for Google Calendar app");
+    }
+
+    const googleAuth = new GoogleAuth({
+      authClient: new OAuth2Client({
+        credentials: {
+          access_token: token.access_token,
+        },
+      }),
+    });
+
+    const meetClient = new ConferenceRecordsServiceClient({
+      auth: googleAuth as ConferenceRecordsServiceClient["auth"],
+    });
+
+    const spacesClient = new SpacesServiceClient({
+      auth: googleAuth as SpacesServiceClient["auth"],
+    });
+
+    const meetingCode = videoCallUrl ? new URL(videoCallUrl).pathname.split("/").pop() : null;
+
+    const spaceInfo = await spacesClient.getSpace({ name: `spaces/${meetingCode}` });
+    const spaceName = spaceInfo[0].name;
+
+    const conferenceRecords = [];
+    for await (const response of meetClient.listConferenceRecordsAsync()) {
```

Comment:
`meetClient.listConferenceRecordsAsync()` returns all conference records which were created by the user. conference record refers to a meeting.